modules/api/src/main/java/org/eclipse/xpanse/api/controllers/OpenTofuMakerWebhookApi.java

```diff
@@ -57,6 +57,23 @@ public void deployCallback(
         openTofuDeploymentResultCallbackManager.deployCallback(UUID.fromString(taskId), result);
     }
 
+    /**
+     * Webhook methods to receive openTofu execution result.
+     */
+    @Tag(name = "Webhook", description = "Webhook APIs")
+    @Operation(description = "Process the execution result after openTofu executes the command "
+            + "line.")
+    @PostMapping(value = "${webhook.tofu-maker.modifyCallbackUri}/{task_id}", produces =
+            MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public void modifyCallback(
```

Comment:
Was just wondering, if we have this webhook, why do we need the deploymentScenario changes in terraform-boot and tofu-maker?
Because when the webhook is called, we already know that the request was for modify. 

If it can work just with the "modify" webhook, then we can later remove the "deploymentScenario" in terraform-boot and tofu-maker.